sdk/common/src/main/java/io/opentelemetry/sdk/internal/ComponentRegistry.java

```diff
@@ -36,14 +38,16 @@ public ComponentRegistry(Function<InstrumentationScopeInfo, V> factory) {
    * any, otherwise creates a new instance and associates it with the given scope.
    */
   public V get(InstrumentationScopeInfo instrumentationScopeInfo) {
+    ScopeKey scopeKey = toScopeKey(instrumentationScopeInfo);
```

Comment:
Note, the identity of Logger, Meter, Tracer is no longer strictly equal to InstrumentationScopeInfo identity because of this [spec PR](https://github.com/open-telemetry/opentelemetry-specification/pull/2789).